README.md

```diff
@@ -79,37 +79,139 @@ return [
 ];
 ```
 
-Then add `SentryMiddleware` to main application middleware set and configure DSN in `config/params.php`. Console errors 
-are captured by default, there is no need to configure anything. 
+If you want to trace Guzzle requests and add Sentry headers to external queries, add the following:
+
+```php 
+GuzzleHttp\Client::class => static function (ContainerInterface $container) {
+    $stack = new HandlerStack();
+    $stack->setHandler(new CurlHandler());
+    $factory = $container->get(GuzzleMiddlewareFactory::class);
+    $middleware = static function (callable $handler) use ($factory): callable {
+        return $factory->factory($handler);
+    };
+
+    $stack->push($middleware);
+
+    return new GuzzleHttp\Client([
+        'handler' => $stack,
+    ]);
+},
+```
+
+
+**Configure:**
+
+Add the following code block to your `params.php` and define DSN. Also you can set "environment" and "release". Good example is to use TAG from gitlab.ci for it.
+```php 
+'yiisoft/yii-sentry' => [
+    'options' => [
+        'dsn' => '',
+        'environment' => 'local', //SENTRY_ENVIRONMENT, //YII_ENV,
+        'release' => 'dev',  //SENTRY_RELEASE, //TAG
+        // @see: https://docs.sentry.io/platforms/php/configuration/options/#send-default-pii
```

Comment:
```suggestion
        // https://docs.sentry.io/platforms/php/configuration/options/#send-default-pii
```

Reply:
This tag has no effect here and it's intended more for linking methods / properties, not for external links.